Test failures from gas price oracle changes in telliot-core

Failures (run locally to reproduce):
```
======================================================================= short test summary info ========================================================================
FAILED tests/feeds/test_active_feeds.py::test_supports_all_active_queries - AssertionError: assert 'gas-price-oracle-example' in {'ampl-legacy': DataFeed(query=Legac...
FAILED tests/feeds/test_gas_price_oracle_feed.py::test_fetch_new_datapoint - assert (None is not None)
FAILED tests/sources/test_gas_price_oracle_source.py::test_fetch_new_datapoint - assert (None is not None)
======================================================== 3 failed, 60 passed, 27 skipped, 2 warnings in 41.26s =========================================================
Terminating local RPC client...
```
Going to skip them until fixed, so the other CI runs don't quit early..

--- a/src/telliot_feed_examples/feeds/gas_price_oracle_feed.py
+++ b/src/telliot_feed_examples/feeds/gas_price_oracle_feed.py
@@ -16,5 +16,6 @@ chain_id = 1
 timestamp = 1648771200  # april 1, 2022 unix time
 
 gas_price_oracle_feed = DataFeed(
-    query=GasPriceOracle(chain_id, timestamp), source=GasPriceOracleSource()
+    query=GasPriceOracle(chain_id, timestamp),
+    source=GasPriceOracleSource(chain_id=chain_id, timestamp=timestamp),
 )

--- a/src/telliot_feed_examples/sources/gas_price_oracle.py
+++ b/src/telliot_feed_examples/sources/gas_price_oracle.py
@@ -1,6 +1,5 @@
+import time
 from dataclasses import dataclass
-from dataclasses import field
-from typing import Dict
 from typing import Optional
 
 import requests
@@ -29,32 +28,33 @@ adapter = HTTPAdapter(max_retries=retry_strategy)
 class GasPriceOracleSource(DataSource[str]):
     """DataSource for GasPriceOracle expected response data."""
 
-    networks: Dict[int, str] = field(
-        default_factory=lambda: {
+    chain_id: int = 1
+    timestamp: int = int(time.time())
+
+    async def fetch_historical_gas_price(
+        self,
+    ) -> Optional[Response]:
+        """Fetches historical gas price data from Owlracle API."""
+
+        networks = {
             1: "eth",
             56: "bsc",
             43114: "avax",
             250: "ftm",
             137: "poly",
             25: "cro",
-            42220: "42220",
+            42220: "one",
             128: "ht",
             1285: "movr",
             122: "fuse",
         }
-    )
-
-    async def fetch_historical_gas_price(
-        self, timestamp: int, chain_id: int
-    ) -> Optional[Response]:
-        """Fetches historical gas price data from Owlracle API."""
 
         url = (
             f"https://owlracle.info/"
-            f"{self.networks[chain_id]}"
+            f"{networks[self.chain_id]}"
             "/history?"
-            f"from={int(timestamp)}"
-            f"&to={int(timestamp) + 100}"
+            f"from={int(self.timestamp)}"
+            f"&to={int(self.timestamp) + 100}"
         )
 
         with requests.Session() as s:
@@ -67,16 +67,14 @@ class GasPriceOracleSource(DataSource[str]):
                 return None
 
     async def fetch_new_datapoint(
-        self, timestamp: int, chain_id: int
+        self,
     ) -> Optional[DataPoint[list[str]]]:
         """Retrieves historical gas prices from Owlracle API.
 
         Returns:
             float gas price in gwei, typically with one decimal place
         """
-        rsp = await self.fetch_historical_gas_price(
-            timestamp=timestamp, chain_id=chain_id
-        )
+        rsp = await self.fetch_historical_gas_price()
         if rsp is None:
             logger.warning("No response from GasPriceOracle API")
             return None, None

--- a/tests/feeds/test_gas_price_oracle_feed.py
+++ b/tests/feeds/test_gas_price_oracle_feed.py
@@ -2,17 +2,15 @@ from datetime import datetime
 
 import pytest
 
-from telliot_feed_examples.feeds.gas_price_oracle_feed import chain_id
 from telliot_feed_examples.feeds.gas_price_oracle_feed import gas_price_oracle_feed
-from telliot_feed_examples.feeds.gas_price_oracle_feed import timestamp
 
 
-@pytest.mark.skip("TODO: Fix")
+# @pytest.mark.skip("TODO: Fix")
 @pytest.mark.asyncio
 async def test_fetch_new_datapoint():
     """Retrieve historical gas price data from source."""
 
-    v, t = await gas_price_oracle_feed.source.fetch_new_datapoint(timestamp, chain_id)
+    v, t = await gas_price_oracle_feed.source.fetch_new_datapoint()
 
     assert v is not None and t is not None
     assert isinstance(v, float)

--- a/tests/feeds/test_gas_price_oracle_feed.py
+++ b/tests/feeds/test_gas_price_oracle_feed.py
@@ -5,7 +5,6 @@ import pytest
 from telliot_feed_examples.feeds.gas_price_oracle_feed import gas_price_oracle_feed
 
 
-# @pytest.mark.skip("TODO: Fix")
 @pytest.mark.asyncio
 async def test_fetch_new_datapoint():
     """Retrieve historical gas price data from source."""

